guides/source/upgrading_ruby_on_rails.md

```diff
@@ -530,6 +530,11 @@ To:
 Rails.application.secrets[:smtp_settings][:address]
 ```
 
+### `render :text` Option is Removed
+
+If your views are rendering using the deprecated `:text` option, replace it with
+an appropriate render option such as `:plain`, `:html`, or `:body`.
```

Comment:
I would mention the direct alternatives eg. `:plain` and `head :ok`.

Reply:
I've updated the PR with your suggestions. I appreciate the review!